test/basic/init_funcs.c

```diff
@@ -0,0 +1,16 @@
+#include "smack.h"
```

Comment:
I think it would be good to have a slightly more realistic example. So maybe introduce two global variables, which get set to something in the initialization functions. Then check whether they were indeed set. I think that would make for a slightly more realistic example.